tests/phpunit/integration/api/actions/beansAddAnonymousAction.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Tests for _beans_add_anonymous_action()
+ *
+ * @package Beans\Framework\Tests\Integration\API\Actions
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Integration\API\Actions;
+
+use Brain\Monkey\Functions;
+use WP_UnitTestCase;
+
+/**
+ * Class Tests_BeansAddAnonymousAction
+ *
+ * @package Beans\Framework\Integration\Unit\API\Actions
+ * @group   unit-integration
+ * @group   api
+ */
+class Tests_BeansAddAnonymousAction extends WP_UnitTestCase {
+
+	/**
+	 * Setup test fixture.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		if ( ! defined( 'BEANS_API_PATH' ) ) {
```

Comment:
Line 30 - 33 shouldn't be needed, Beans should be loading by default from the bootstrap file.

Reply:
"Doh" moments crept in....it's an integration test and not a unit test. LOL  

Latest commit removed the `setUp()` method.